### PR TITLE
Cap 5 support

### PR DESCRIPTION
**Pull request by qliqdev**
Added support for Capacitor v5

### CODE REVIEW

**Comment by johnborges** [comment]
@qliqdev did you mean to close this?

**Comment by qliqdev** [comment]
i've closet this PR, cuz you have already migrated to v5

**Comment by johnborges** [comment]
My PR was for main branch which doesn't include Android platform at the moment. It would be nice if we can merge your PR into [capacitor-community:android-support](https://github.com/capacitor-community/app-icon/tree/android-support).

**Comment by qliqdev** [comment]
sure. I'll fix conflicts now

**Review by johnborges — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@capacitor-community/app-icon",
-  "version": "3.1.0-beta.1",
+  "version": "5.0.0",
```
[the remark this answers is not in the document]
Comment (dabaaaz) [reply package.json:3]: Or 5.0.0-beta.1 ?

**Pull request merged**